File: 01恋上数据结构与算法(Java)/Java版/j07队列/testQueue.java
/*************************************************************************
> File Name       : testQueue.java
> Author          : Harold
> Mail            : 2106562095@qq.com
> Github          : www.github.com/Haroldcc
> Created Time    : 2020年03月03日  20时00分41秒
************************************************************************/

package j07队列;

/**
 * testQueue
 */
public class testQueue {

    public static void main(String[] args) {
        Queue<Integer> queue = new Queue<>();

        queue.enQueue(11);
        queue.enQueue(22);
        queue.enQueue(33);
        queue.enQueue(44);

        while (!queue.isEmpty()) {
            System.out.println(queue.deQueue());
        }
    }
}

File: 02DataStructuresAndAlgorithms/linearLIstStructures/testQueue.cpp

/*************************************************************************
> File Name       : testQueue.cpp
> Author          : Harold
> Mail            : 2106562095@qq.com
> Github          : www.github.com/Haroldcc
> Created Time    : 2020年03月05日  10时29分46秒
************************************************************************/
#include "queue.h"
#include <iostream>

using namespace std;

int main()
{
    queue<int> qu;
    qu.enQueue(11);
    qu.enQueue(22);
    qu.enQueue(33);

    std::cout << qu.top() << std::endl;
    qu.deQueue();

    std::cout << qu.top() << std::endl;
    qu.deQueue();

    std::cout << qu.top() << std::endl;
    qu.deQueue();

    std::cout << qu.top() << std::endl;
    qu.deQueue();

    std::cout << qu.top() << std::endl;
    qu.deQueue();

    return 0;
}